aip/general/0001.md

```diff
@@ -3,17 +3,13 @@ aip:
   id: 1
   state: approved
   created: 2018-08-20
-  updated: 2019-05-04
-  type: process
-js:
-  - /assets/js/graphviz/viz.js
-  - /assets/js/graphviz/lite.render.js
-  - /assets/js/aip/aip-graphviz.js
-permalink: /1
-redirect_from:
-  - /01
-  - /001
-  - /0001
+  placement:
+    category: meta
+    order: 10
+  js_scripts:
```

Comment:
As long as we are doing a pretty massive refactor, these could be moved to be default behavior. Or implement something similar to the `_config.yaml` where these also could have been moved to.